components/raftstore/src/store/snap.rs

```diff
@@ -1959,6 +1960,28 @@ impl TabletSnapManager {
         let prefix = format!("{}_{}", SNAP_GEN_PREFIX, key);
         PathBuf::from(&self.base).join(&prefix)
     }
+
+    pub fn get_final_name_for_recv(&self, key: &TabletSnapKey) -> PathBuf {
+        let prefix = format!("{}_{}", SNAP_REV_PREFIX, key);
+        PathBuf::from(&self.base).join(&prefix)
+    }
+    pub fn get_tmp_name_for_recv(&self, key: &TabletSnapKey) -> PathBuf {
+        let prefix = format!("{}_{}{}", SNAP_REV_PREFIX, key, TMP_FILE_SUFFIX);
+        PathBuf::from(&self.base).join(&prefix)
+    }
+
+    pub fn delete_snapshot(&self, key: &TabletSnapKey) -> bool {
+        let path = self.get_tablet_checkpointer_path(key);
```

Comment:
Let's fix some name usage while you are at it. The checkpointer is a stateless helper struct that can be used to create checkpoints at many given checkpoint_path-s. Here we should use `get_tablet_checkpoint_path`.